spec/apress/moysklad/presenters/assortment_spec.rb

```diff
@@ -12,60 +12,70 @@
     let(:row) { Oj.load File.new('spec/fixtures/assortment.json'), symbol_keys: true }
 
-    subject { presenter.expose(row) }
+    subject do
+      VCR.use_cassette 'read_images' do
+        presenter.expose(row)
+      end
+    end
 
     it 'return hash' do
       is_expected.to eq __line__: 1,
                         __column__: 1,
 
-                        id: '1a3cc760-eaf6-11e7-6b01-4b1d001d43a0',
-                        code: '4547',
-                        accountId: '5dde328f-99e6-11e7-7a69-971100003000',
-                        externalCode: 'k8wwtsTajfpNw9MiHwbTq1',
+                        id: '875d7130-d31d-11e9-0a80-025a002970c1',
+                        code: '00001',
+                        accountId: '56bd4617-d31d-11e9-0a80-025300024b0d',
+                        externalCode: 'wyFpOzV1gcOJVnGyvJ--m1',
 
-                        article: 'К-001',
-                        name: 'Конина',
-                        description: 'Вкусная, полезная',
+                        article: '290307',
+                        name: 'Платье со сборкой на боку',
+                        description: "Состав: 90% тенсел, 10% лен\nСтрана-производитель: КИТАЙ",
 
-                        minPrice: 20_000.0,
+                        minPrice: {value: 500_000.0},
                         pathName: 'Первый уровень/Второй уровень',
-                        version: 1,
 
                         stock: 0.0,
                         reserve: 0.0,
                         inTransit: 0.0,
                         quantity: 0.0,
 
                         archived: false,
-                        updated: '2017-12-27 16:11:07',
+                        updated: '2020-09-14 11:01:21.392',
 
-                        modificationsCount: 1,
                         weight: 1_000.0,
-                        volume: 500.0,
-                        vat: 0,
-                        effectiveVat: 0,
+                        volume: 1_000.0,
+                        vat: 10,
+                        effectiveVat: 10,
 
                         meta: {type: 'product'},
-                        buyPrice: {value: 10_000.0},
+                        buyPrice: {value: 450_000.0},
 
                         salePrices: [
-                          {value: 0.0, priceType: 'опт'},
-                          {value: 25_000.0, priceType: 'Розничная цена'}
+                          {
+                            priceType: {
+                              externalCode: "cbcf493b-55bc-11d9-848a-00112f43529a",
+                              id: "56fb6e43-d31d-11e9-0a80-01810016c891",
+                              meta: {
+                                href: "https://api.moysklad.ru/api/remap/1.2/context/companysettings/pricetype/56fb6e43-d31d-11e9-0a80-01810016c891",
+                                mediaType: "application/json",
+                                type: "pricetype"
+                              },
+                              name: "Цена продажи"
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.